packages/mdc-select/mdc-select.scss

```diff
@@ -264,3 +264,8 @@
 // with CSS. Use the font-size rule instead.
 
 // postcss-bem-linter: end
+
+.mdc-select-scroll-lock {
+  height: 100vh;
```

Comment:
why do you need to set the hight? Is `overflow: hidden` not enough to remove the scroll bar?

Reply:
It's just like that in the drawer implementation

Reply:
I did some digging into the drawer (and dialog for that matter) and we're 99% sure that `height: 100vh` is unnecessary. 

So, @roncapat, go ahead and remove it here too. (#1610 and #1612 for the changes we're making to drawer and dialog)

Reply:
IMHO, if we remove `height: 100vh`, scroll will be permitted and so select menu overlay won't follow the base selector on page. (I've done the changes you requested, but I have to test the behaviour).